cli/src/commands/extensions/extension.rs

```diff
@@ -122,10 +139,13 @@ impl Extension {
         Extension::try_from(extension_path(name)?)
     }
 
+    pub fn path(&self) -> Result<PathBuf, std::io::Error> {
```

Comment:
```suggestion
    pub fn path(&self) -> Result<PathBuf, io::Error> {
```

Maybe we'd also like to add a doc comment that canonicalization is performed, just in case someone calls it and is curious.